pandas/tests/io/json/test_normalize.py

```diff
@@ -277,6 +277,56 @@ def test_missing_field(self, author_missing_data):
         expected = DataFrame(ex_data)
         tm.assert_frame_equal(result, expected)
 
+    def test_records_path_with_nested_data(self):
```

Comment:
Can you simplify this test via parametrization? Probably OK to parametrize arguments of `max_level` and `expected`